File: edudoc/src/API/AdminPortal/Providers/DTOs/ProviderDTO.cs
using Model;
using Model.DTOs;

namespace API.Providers.DTOs
{
    public class ProviderDTO
    {
        public User User
        {
            get; set;
        }

        public AuthUserDTO AuthUser
        {
            get; set;
        }

        public Provider Provider
        {
            get; set;
        }

        public int OldEmploymentTypeId
        {
            get; set;
        }

        public int UserTypeId
        {
            get; set;
        }

        public bool? SendEmail
        {
            get; set;
        }
    }
}

File: edudoc/src/app/admin-portal/providers/libraries/dtos/provider.dto.ts
import { IAuthUserDTO } from '@model/interfaces/custom/auth-user.dto';
import { IProvider } from '@model/interfaces/provider';
import { IUser } from '@model/interfaces/user';

export interface IProviderDTO {
    AuthUser: IAuthUserDTO;
    User: IUser;
    Provider: IProvider;
    SendEmail: boolean;
    OldEmploymentTypeId: number;
    UserTypeId: number;
}
